kurl_util/cmd/yamlutil/main.go

```diff
@@ -99,20 +101,71 @@ func retrieveField(filePath, yamlPath string) {
 	}
 }
 
+func jsonField(filePath, jsonPath string) {
+	configuration := readFile(filePath)
+
+	var parsed interface{}
+
+	err := yaml.Unmarshal(configuration, &parsed)
+
+	if err != nil {
+		log.Fatalf("error: %v", err)
+	}
+
+	fields := strings.Split(jsonPath, ".")
+
+	// get the specified field
+	for _, field := range fields {
+		concrete := parsed.(map[interface{}]interface{})
+		parsed = concrete[field]
```

Comment:
should you do some safety checking here that the field exists with a nice error message?

Reply:
sure, makes sense